scripts/extract_strings.sh

```diff
@@ -76,21 +76,53 @@ set -x
 
 
 # Gettext extraction
-(cd $CONTENT_DIR && npx grunt l10n-extract)
-cp -r $CONTENT_DIR/locale/templates/* $L10N_DIR/locale/templates
 
-# see https://github.com/mozilla/zippy/blob/180f746cd84cb3e409c76649e810485acb4512dc/locale/omg_new_l10n.sh#L90
-# also see https://bugzilla.mozilla.org/show_bug.cgi?id=1168488
-msgfilter -i $L10N_DIR/locale/sr/LC_MESSAGES/client.po -o $L10N_DIR/locale/sr_Latn/LC_MESSAGES/client.po recode-sr-latin
-msgfilter -i $L10N_DIR/locale/sr/LC_MESSAGES/server.po -o $L10N_DIR/locale/sr_Latn/LC_MESSAGES/server.po recode-sr-latin
+# Extract the hash of the pot files content without the POT-Creation-Date line.
+# This is used to avoid copying over the templates if the actual content
+# was not updated by the l10n-extract script
+OLD_HASHES=$(python $L10N_DIR/scripts/pot_checksum.py $L10N_DIR/locale/templates/)
```

Comment:
I'm sure this could have been done via Bash, but I didn't want to go down a rabbit hole of commands, and the Python script is pretty easy to go through, without requiring extra packages.